src/config.ts

```diff
@@ -4,6 +4,8 @@ const config = {
   REACT_APP_ROOT_ID: process.env.REACT_APP_ROOT_ID ?? '',
   // Ued to list all files in a fast way.
   REACT_APP_ROOT_DRIVE_ID: process.env.REACT_APP_ROOT_DRIVE_ID ?? '',
+
+  fields: 'nextPageToken, files(name, id, parents, mimeType, modifiedTime, createdTime, lastModifyingUser(displayName, photoLink), iconLink, webViewLink, shortcutDetails, capabilities/canAddChildren)',
```

Comment:
Fields has been updated, please use new fields, for example, as you can see, `capabilities(canAddChildren, canTrash, canRename)` are missing.